extensions/VisualEditor/modules/ve-mw/ui/pages/ve.ui.MWParameterPage.js

```diff
@@ -159,6 +159,15 @@ ve.ui.MWParameterPage = function VeUiMWParameterPage( parameter, name, config )
 	} else {
 		this.infoButton.getPopup().$body.append( this.$description );
 	}
+
+	// TODO no!
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)